unordered-containers.cabal

```
@@ -42,6 +42,14 @@ flag debug
   description:  Enable debug support
   default:      False
 
+flag noSse42
+  description: Instruct GHC to *not* use -msse4.2. Typical use case is to allow
+               older architectures to use unordered-containers. Only relevant
+               for x86; expect performance regressions on newer architectures
+               when enabled.
+  default:     False
```

```suggestion
flag sse42
  description: Instruct GHC to *not* use -msse4.2. Typical use case is to allow
               older architectures to use unordered-containers. Only relevant
               for x86; expect performance regressions on newer architectures
               when enabled.
  default:     True
```
I think it makes more sense to call the flag `sse42` and avoid negations in the name (you can disable flags by using `-sse42`).

---

Won't this default value disable the flag for a user on x86 who does not pass anything? Since now the conditional will evaluate to `False && True`? I thought that we wanted the flag turned on for x86_64 by default (ie, without the user needing to do anything), and then provide an option to disable it.

---

Great. I just wanted to make sure we agreed about the default-ness of the flag